convert2rhel/toolopts.py

```diff
@@ -431,6 +433,15 @@ def warn_on_unsupported_options():
         utils.ask_to_continue()
 
 
+def _log_command_used():
+    """We want to log the command used for convert2rhel to make it easier to know what command was used
+    when debugging the log files. Since we can't differenciate between the handlers we log to both stdout
+    and the logfile
+    """
```

Comment:
Small typo
```suggestion
    """We want to log the command used for convert2rhel to make it easier to know what command was used
    when debugging the log files. Since we can't differentiate between the handlers we log to both stdout
    and the logfile
    """
```

Reply:
Good catch!